src/services/dynamicsDal.service.js

```diff
@@ -93,7 +93,8 @@ module.exports = class DynamicsDalService {
           console.log(`Dynamics query response: Status Code: ${response.statusCode} Message: ${response.statusMessage}`)
 
           if (response.statusCode === 204) {
-            resolve()
+            const id = /\((.*?)\)/.exec(response.headers['odata-entityid'])[1]
```

Comment:
Might be helpful to have a comment explaining what the regex is doing.

Reply:
Yeah, cause I understand and didn't just crib it off the interwebs 😳 .

But will add something.